### ISSUE TITLE
It isn't possible to make a cow in a jar because of Neruina killing the anvil

### ISSUE THREAD

**Issue by dittymargoo** [The-Nexus-Project/Limitless-7#83]
### What happened?

This just happened minutes ago:  I attempted to make a cow in a jar and got this message: "Neruina: [!] Caught Ticking Entity [Falling Anvil]...".  So I switched to creative and tried over and over but I still get the same outcome.
![Ticking Entity Falling Anvil TNP7](https://github.com/The-Nexus-Project/Limitless-7/assets/55962902/4b00537f-99d1-4d84-b6e0-b8cf3288bde8)

### Pack Version

Version: 0.35.1

### To Reproduce

1. Place an empty milk jar at the bottom of a hole
2. Get a cow (mine's a white one) in a hole
3. Drop an anvil on the cow
4. Anvil disappears and Ticking Entity message appears.

### Are you using OptiFine or Oculus?

- [ ] Using OptiFine
- [ ] Using Oculus
- [X] No. None of them. Unmodified.

### Are you on a Server?

- [ ] Yes.
- [X] No.

### Relevant log output / crash report.

```shell
No crash to report.
```

### Anything else to add?

I love love love this pack.  Thanks for all the hard work it is most appreciated!

**Comment by dittymargoo** [comment]
Was wondering if this had anything to do with issue #68?? Maybe, but either way I'm off to try and disable Neruina for just this and then re-enable it.

**Comment by dittymargoo** [comment]
Nope, didn't work, game just crashes (I tried twice) and even messed with the config json file settings.  So looks like I'll have to cheat in a 'cow in a jar' for now and leave the problem solving to someone more capable than me.

**Comment by Gbergz** [comment]
Update modpack to 0.37.1, fresh install. Try again.

**Comment by dittymargoo** [comment]
Thank you, will do.  I hadn't even realised there was an update...yeah, I need to do something about my eyesight.
